rows_picker.go

```diff
@@ -0,0 +1,1018 @@
+package sqlmw
```

Comment:
this file should have an appropriate generation comment at the top of it

Reply:
sure, but the comment should indiciate that the file was generated, the command that generated it and not to edit the file manually